File: src/test/java/info/mastera/leetcode/intersectionOfTwoArrays349/SolutionTest.java
package info.mastera.leetcode.intersectionOfTwoArrays349;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class SolutionTest {

    @Test
    public void test() {
        Assertions.assertArrayEquals(new int[]{2}, new Solution().intersection(new int[]{1, 2, 2, 1}, new int[]{2, 2}));
        Assertions.assertArrayEquals(new int[]{4, 9}, new Solution().intersection(new int[]{4, 9, 5}, new int[]{9, 4, 9, 8, 4}));
    }
}

File: src/test/java/info/mastera/leetcode/reverseString344/SolutionTest.kt
package info.mastera.leetcode.reverseString344

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

class SolutionTest {

    @Test
    fun test() {
        var workingArray = "hello".toCharArray()
        Solution().reverseString(workingArray)
        assertArrayEquals("olleh".toCharArray(), workingArray)

        workingArray = "Hannah".toCharArray()
        Solution().reverseString(workingArray)
        assertArrayEquals("hannaH".toCharArray(), workingArray)
    }
}